doc/docusaurus/docs/auction-smart-contract/end-to-end/mint.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 15
+---
+
+# Minting the Token to be Auctioned
+
+Before we can start the auction, we need to mint a token to be auctioned.
+On Cardano, each token has a currency symbol and a token name.
```

Comment:
```suggestion
On Cardano, each token is uniquely identified by its currency symbol and a token name.
```

Reply:
That doesn't sound accurate, since currency symbol + token name uniquely identify a class of tokens, but there can be more than one tokens of that class.

Reply:
Right. The word "token" is overloaded. I meant "class of tokes". My goal with this change was to avoid new developers' common confusion about the currency symbol being enough to define/identify the token. Else, the original was fine. 

Being mindful of the context, does this sound better?
```suggestion
To do so, we first must determine the token's currency symbol and name.
```